pkg/controller/core/localqueue_controller.go

```diff
@@ -204,6 +264,29 @@ func (r *LocalQueueReconciler) Update(e event.TypedUpdateEvent[*kueue.LocalQueue
 	return true
 }
 
+func (r *LocalQueueReconciler) ReconcileConsumedUsage(ctx context.Context, lq *kueue.LocalQueue, cqName kueue.ClusterQueueReference) error {
+	halfLifeTimeSeconds := r.admissionFSConfig.UsageHalfLifeTime.Seconds()
+	oldUsage := lq.Status.FairSharingStatus.AdmissionFairSharingStatus.ConsumedResources
+	// reset usage to 0 if halfLife is 0
+	if halfLifeTimeSeconds == 0 {
+		lq.Status.FairSharingStatus.AdmissionFairSharingStatus.ConsumedResources = resource.MulByFloat(oldUsage, 0)
+		lq.Status.FairSharingStatus.AdmissionFairSharingStatus.LastUpdate = metav1.NewTime(r.clock.Now())
+		return r.client.Status().Update(ctx, lq)
+	}
```

Comment:
During this time, workloads will still be admitted, but their usage won't be counted, right? Is this something we need to handle (disabling scheduling)?